src/zeekygen/zeekygen.bif

```diff
@@ -33,6 +33,26 @@ function get_identifier_comments%(name: string%): string
 	return comments_to_val(d->GetComments());
 	%}
 
+## Retrieve the declaring script associated with an identifier
+## (e.g. a variable or type).
+##
+## name: a script-level identifier
+##
+## Returns: declaring script associated with *name*. If *name* is not a known
+##          identifier, an empty string is returned.
```

Comment:
I get an empty string for builtins, e.g, `get_identifier_declaring_script("count")`, so either the documentation or the implementation should be tweaked (maybe more interesting with language evolution in mind, same result for `get_identifier_declaring_script("print")`).

Naively I would assume that being able to get results with builtins as well would be useful. I wonder whether it wouldn't make sense e.g., return a richer type, e.g., an instance of

```zeek
type DeclaringScript: record {
	path: string &optional;
};
```

I didn't check, but I suspect `get_record_field_declaring_script` could show the same issue for types without matching scriptland record.

Reply:
> I get an empty string for builtins, e.g, get_identifier_declaring_script("count"), so either the documentation or the implementation should be tweaked (maybe more interesting with language evolution in mind, same result for get_identifier_declaring_script("print")).
> Naively I would assume that being able to get results with builtins as well would be useful.

Hmm, so that was mostly copied from the `_comments` bif. Maybe the fix is to include the `script-level` part before the `identifier` in the `Returns:` documentation?

One could probably explicitly populate zeekygen info for builtins if that was interesting, but it's not clear what the declaring script would be.